actions/nuxeo-action-element-styles.html

```diff
@@ -0,0 +1,51 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../../polymer/polymer.html">
+
+<!--
+  Styles module to be used by action elements.
+
+  Custom property | Description | Default
+  ----------------|-------------|----------
+  `--nuxeo-action-element` | Mixin applied to the action element | { display: inline-block; }
+  `--nuxeo-action-element-show-label` | Mixin applied to the action element when showing label | {}
+  `--nuxeo-action-element-label` | Mixin applied to action label | {}
+-->
+<dom-module id="nuxeo-action-element-styles">
```

Comment:
nit: not a big fan of having `element` as ultimately everything is about elements... perhaps `nuxeo-action-button-styles` or something

Reply:
Don't really like the button there, because an action doesn't have to be a button, hence the element. We could have `nuxeo-action-styles`, WDYT?

Reply:
works for me, it's just that every action today is `blah-button` :P

Reply:
OTOH, `nuxeo-action-styles` makes it much harder to find good names for the mixins/variables. Will consider the "button" after all. :)